namespaces/cloud-platform-live-0.k8s.integration.dsd.io/support-labelling-webhook-dev/00-namespace.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: support-labelling-webhook-dev
+  labels:
+    name: support-labelling-webhook
+    cloud-platform.justice.gov.uk/business-unit: cloud-platform
+    cloud-platform.justice.gov.uk/is-production: false
```

Comment:
This might break, I believe it has to be a string, `"false"`, unless kubernetes automatically converts it.

Reply:
Okay let me change that. I did read somewhere that it was generally safer to put all the values as strings unless you explicitly wanted them to be e.g. numbers or whatever.